enterprise/api_client/lms.py

```diff
@@ -469,6 +469,17 @@ def get_course_grade(self, course_id, username):
 
         raise HttpNotFoundError('No grade record found for course={}, username={}'.format(course_id, username))
 
+    @JwtLmsApiClient.refresh_token
+    def get_course_assessment_grades(self, course_id, username):
+        """
+        Placeholder
+        """
+        results = self.client.gradebook(course_id).get(user_contains=username).get('results')
```

Comment:
Writing up here because we did offline work- TLDR platform's gradebook API currently has a bug and `500's` when specifically querying with `username`. We were able to find out exactly why this happens, but lack the context within the gradebook API to suggest a fix at this point. I've notified `masters-grades` but their response time seems a tad slow so we're going to continue with this line until we can get some more info.

Reply:
is there a tracking ticket for this? if we need to make a change here after that it may be worth noting here or even in the code... otherwise okie

Reply:
So far no one has responded to my messages in `masters-grades`

Reply:
we got a response! And hopefully a fix soon 🤞